### PR TITLE
Fixes issue while closing many tabs.

### LINKED ISSUE #22399: Godot does't recognize if is visible scene tab when closing it
**Godot version:**
3.1 399910d

**OS/device including version:**
Windows 10

**Issue description:**
Godot should check if scenes tab are visible when closing some of it.

**Steps to reproduce:**
jfile

### PR DESCRIPTION
**Pull request by twinderw**
Fixes #22399.

### CODE REVIEW

**Comment by twinderw** [comment]
Fixing the build issues now

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -153,6 +153,12 @@ void EditorNode::_update_scene_tabs() {
 
 	scene_tabs->set_current_tab(editor_data.get_edited_scene());
 
+	// ensures that a tab will still display if the last visible tab was closed.
+	if (editor_data.get_edited_scene_count() < previous_num_open) {
```
[the remark this answers is not in the document]
Comment (twinderw) [reply editor/editor_node.cpp]: @akien-mga Good catch. I just put the initialization in with some other variables, but if there's a better place to put it please let me know and I'll move it.

**Comment by akien-mga** [comment]
Could you squash the commits together? See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

**Comment by twinderw** [comment]
Done, I think.

File: editor/editor_node.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (akien-mga) [reply editor/editor_node.cpp]: It's still used uninitialized in the current version.

**Comment by akien-mga** [comment]
Moving to 3.2 milestone as we're about to enter the beta stage and *release freeze* for Godot 3.1. Only bug fixes against issues of the 3.1 milestone (or enhancements to core features of the 3.1 roadmap) will be considered until 3.1-stable is released.

Note: If it's merged after 3.1-stable is released, this change may be considered for cherry-picking into a later 3.1.x release.

File: editor/editor_node.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (akien-mga) [reply editor/editor_node.cpp]: Bump.

**Comment by akien-mga** [comment]
Closing as there are issues which are not being addressed, so it can't be merged.

**Pull request closed without merging**